### PR TITLE
Add language attribute to html element

### PR DESCRIPTION
**Pull request by rixx**

### CODE REVIEW

**Comment by rixx** [comment]
Ah, I forgot about pretix' non-standard language codes. Yes, you want standardised language attributes, and choose the shortest attribute applicable (favoring the "de" over the subtag+region subtag combination "de-DE". At the same time, the region subtag in "pt-BR" or in "nl-BE" is required and cannot be left out.) 
As pretix currently seems to use shortened versions wherever possible, it seems to me the only thing we need to strip is the custom "informal" subtag.

File: src/pretix/control/forms/filter.py [hunk 1]
```diff
@@ -739,7 +739,6 @@ class VoucherFilterForm(FilterForm):
         label=_('Search voucher'),
         widget=forms.TextInput(attrs={
             'placeholder': _('Search voucher'),
-            'autofocus': 'autofocus'
```
Comment (raphaelm) [inline src/pretix/control/forms/filter.py]: Did you remove this on purpose?

Comment (rixx) [reply src/pretix/control/forms/filter.py]: I did, as the commit message states: [control] Remove second autofocus element on voucher list

Comment (raphaelm) [reply src/pretix/control/forms/filter.py]: Sorry, I didn't look at the commit message.

**Comment by rixx** [comment]
Sorry, I don't think I'll find the time to finish this any time soon.

**Pull request closed without merging**